frontend/src/components/accounts/Accounts.tsx

```diff
@@ -4,42 +4,138 @@ import AccountsApi, * as A from "../../libraries/explorer-wamp/accounts";
   };
 
   _accountApi: AccountsApi | null;
+  timer: ReturnType<typeof setTimeout> | null;
+  _accountLoader: Element | null;
 
   constructor(props: Props) {
     super(props);
     this._accountApi = null;
+    this.timer = null;
+    this._accountLoader = null;
+  }
+
+  componentDidMount() {
+    this._accountApi = new AccountsApi();
+    document.addEventListener("scroll", this._onScroll);
+    this.timer = setTimeout(this.regularFetchInfo, 0);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener("scroll", this._onScroll);
+    clearTimeout(this.timer!);
+    this.timer = null;
   }
 
+  regularFetchInfo = async () => {
+    await this.fetchAccounts();
+    if (this.timer !== null) {
+      this.timer = setTimeout(this.regularFetchInfo, 10000);
+    }
+  };
+
   fetchAccounts = async () => {
     if (this._accountApi === null) {
       this._accountApi = new AccountsApi();
     }
-    const accounts = await this._accountApi.getAccounts();
+    let accounts;
+    if (this.state.accounts.length === 0) {
+      accounts = await this._accountApi.getAccounts(this.props.limit);
+    } else {
+      accounts = await this._accountApi.getAccounts(this.state.accounts.length);
+    }
     this.setState({ accounts });
   };
 
-  componentDidMount() {
-    this.fetchAccounts();
-  }
+  _isAtBottom = () => {
+    return (
+      this._accountLoader &&
+      this._accountLoader.getBoundingClientRect().bottom <= window.innerHeight
+    );
+  };
+
+  _onScroll = async () => {
+    this._accountLoader = document.querySelector("#account");
+    const bottom = this._isAtBottom();
+    if (bottom) {
+      document.removeEventListener("scroll", this._onScroll);
+      await this._loadAccounts();
+      this.setState({ loading: false });
+      document.addEventListener("scroll", this._onScroll);
+    }
+  };
+
+  _loadAccounts = async () => {
+    const count = await this._getLength();
+    if (count) {
+      if (count <= this.state.accounts.length) {
+        this.setState({ loading: false });
+      } else {
+        await Promise.all([
+          this.setState({ loading: true }),
+          this._addAccounts()
+        ]);
+      }
+    }
+  };
+
+  _addAccounts = async () => {
+    if (this._accountApi === null) {
+      this._accountApi = new AccountsApi();
+    }
+    const lastIndex = this.state.accounts[0].timestamp;
+    const accounts = await this._accountApi.getAccounts(
+      this.props.limit,
+      lastIndex
+    );
+    const _accounts = this.state.accounts;
+    const Accounts = _accounts.concat(accounts);
+    this.setState({ accounts: Accounts });
+  };
+
+  _getLength = async () => {
+    if (this._accountApi === null) {
+      this._accountApi = new AccountsApi();
+    }
+    const count = await this._accountApi.getAccountLength();
+    return count;
+  };
 
   render() {
-    const { accounts } = this.state;
-    if (accounts === null) {
+    const { accounts, loading } = this.state;
+    if (accounts === []) {
```

Comment:
Sadly, `[] === []` is `false` is JS world. In fact, since you have added the `loading` state variable, we can use it here.

In fact, we can completely remove this `if` block as it should work just fine without it.